src/controller/SetUpCodePairer.cpp

```diff
@@ -104,26 +104,15 @@ CHIP_ERROR SetUpCodePairer::Connect()
         if (ShouldDiscoverUsing(RendezvousInformationFlag::kBLE))
         {
             CHIP_ERROR err = StartDiscoveryOverBLE();
-            if (err != CHIP_NO_ERROR)
+            if ((err != CHIP_NO_ERROR) && (err != CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE))
             {
                 ChipLogError(Controller, "Failed to start discovery over BLE: %" CHIP_ERROR_FORMAT, err.Format());
             }
         }
-
-        if (ShouldDiscoverUsing(RendezvousInformationFlag::kSoftAP))
-        {
-            CHIP_ERROR err = StartDiscoveryOverSoftAP();
-            if (err != CHIP_NO_ERROR)
-            {
-                ChipLogError(Controller, "Failed to start discovery over SoftAP: %" CHIP_ERROR_FORMAT, err.Format());
-            }
-        }
         if (ShouldDiscoverUsing(RendezvousInformationFlag::kWiFiPAF))
         {
-            ChipLogProgress(Controller,
-                            "WiFi-PAF: has RendezvousInformationFlag::kWiFiPAF or has no discovery capabilities bitmask");
             CHIP_ERROR err = StartDiscoveryOverWiFiPAF();
-            if (err != CHIP_NO_ERROR)
+            if ((err != CHIP_NO_ERROR) && (err != CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE))
```

Comment:
Again, knowing "your device was not discovered because it only advertises over PAF and PAF is unsupported in the commissioner" is in fact useful.

Reply:
Fixed